<p>Avoid returning java.util.Date type, this class is deprecated and using other classes such as LocalDate is preferred</p>
<h2>Noncompliant Code Example</h2>
<pre>
  public Date methodReturningDate(){
    // Code here
  }
</pre>
<h2>Compliant Solution</h2>
<pre>
  public LocalDate methodReturningDate(){
    // Code here
  }
</pre>
